### PR TITLE
Don't measure coverage on generated code

### PR DESCRIPTION
**Pull request by MrCreosote**

### CODE REVIEW

**Comment by MrCreosote** [comment]
Hmm, not sure why the client is still showing up

File: build.xml [hunk 1]
```diff
@@ -219,11 +219,51 @@ java -cp ${jar.absolute.path}:${lib.classpath} kbasesearchengine.tools.SearchToo
       <structure name="KBaseSearchEngine Unit Tests">
         <classfiles>
           <fileset dir="${classes}">
+            <!-- there must be a better way to ignore the generated code-->
```
Comment (arfathpasha) [inline build.xml]: You get a little bit of DRY with a patternset

<patternset id="exclude-from-coverage">
    <exclude name="kbasesearchengine/AccessFilter.*"/>
    <exclude name="kbasesearchengine/GetObjectsInput.*"/>
    <exclude name="kbasesearchengine/GetObjectsOutput.*"/>
    <exclude name="kbasesearchengine/KBaseSearchEngineClient.*"/>
    <exclude name="kbasesearchengine/KeyDescription.*"/>
    <exclude name="kbasesearchengine/ListTypesInput.*"/>
    <exclude name="kbasesearchengine/ListTypesOutput.*"/>
    <exclude name="kbasesearchengine/MatchFilter.*"/>
    <exclude name="kbasesearchengine/MatchValue.*"/>
    <exclude name="kbasesearchengine/ObjectData.*"/>
    <exclude name="kbasesearchengine/Pagination.*"/>
    <exclude name="kbasesearchengine/PostProcessing.*"/>
    <exclude name="kbasesearchengine/SearchObjectsInput.*"/>
    <exclude name="kbasesearchengine/SearchObjectsOutput.*"/>
    <exclude name="kbasesearchengine/SearchTypesInput.*"/>
    <exclude name="kbasesearchengine/SearchTypesOutput.*"/>
    <exclude name="kbasesearchengine/SortingRule.*"/>
    <exclude name="kbasesearchengine/TypeDescriptor.*"/>
    <exclude name="us/kbase/**"/>
    <exclude name="**/test/**" />
  </patternset>

  <taskdef uri="antlib:org.jacoco.ant" resource="org/jacoco/ant/antlib.xml"
           classpathref="jacoco.classpath"/>

  <target name="test-report"
          depends="test"
          description="run all tests and generate test report">
    <jacoco:report>
      <executiondata>
        <file file="${test.reports.dir}/jacoco.exec"/>
      </executiondata>
      <structure name="KBaseSearchEngine Unit Tests">
        <classfiles>
          <fileset dir="${classes}">
            <patternset refid="exclude-from-coverage"/>
          </fileset>
        </classfiles>
        <sourcefiles encoding="UTF-8">
          <fileset dir="${src}">
            <patternset refid="exclude-from-coverage"/>
          </fileset>
        </sourcefiles>
      </structure>
      <html destdir="${test.reports.dir}/html"/>
      <csv destfile="${test.reports.dir}/coverage-report.csv"/>
      <xml destfile="${test.reports.dir}/coverage-report.xml"/>
    </jacoco:report>
  </target>

Comment (MrCreosote) [reply build.xml]: I just see blank text

Comment (MrCreosote) [reply build.xml]: editing shows it, duh

**Comment by MrCreosote** [comment]
I'm still not sure why the client is showing up, but I don't want to spend any more time on this.

**Pull request merged**